src/llama.cpp

```diff
@@ -2237,6 +2239,7 @@ struct llama_hparams {
     uint32_t ssm_d_inner = 0;
     uint32_t ssm_d_state = 0;
     uint32_t ssm_dt_rank = 0;
+    bool ssm_dt_b_c_rms = false;
```

Comment:
This new field should also be handled in the `operator!=` below.

Maybe also add a line for it in `llm_load_print_meta`.

Reply:
Thanks, done !

Reply:
You might want to `git reset` before your last commit and `git push --force-with-lease` that, because I think that last commit accidentally removed `src/llama.cpp` entirely.

Reply:
ah yes good point, let me fix that

Reply:
This time we should be safe .. Somehow when pushing with VS code I get random lines being deleted (maybe due to the file having a lot of lines), it works fine when using the command line

Reply:
That's good to know. Also, the format for `bool` values is `%d`, because they are printed as integers.

Otherwise I get
```
src/llama.cpp:5915:24: warning: format ‘%s’ expects argument of type ‘char*’, but argument 4 has type ‘int’ [-Wformat=]
```

Reply:
ok let me fix that

Reply:
Done https://github.com/ggerganov/llama.cpp/pull/9074/commits/7aeccbb7ddd0cbf349140fe7d54917d8981edde4